code_folder/C0_data_processing.py

```diff
@@ -347,7 +422,7 @@ def define_dso_sinks_and_sources(dict_values, sector, dso):
         helpers.define_sink(
             dict_values,
             dso + "_feedin",
-            -dict_values["energyProviders"][sector][dso]["feedin_tariff"]["value"],
+            dict_values["energyProviders"][sector][dso]["feedin_tariff"],
```

Comment:
Is feed-in tariff not defined like
"feed-in tariff": {'value': scalar/list, 'unit': "str"}?

Reply:
no, I suggested keeping the format agreed. If feed-in tariff is a timeseries, a filename will be provided, and in that case there will be no value.

Reply:
This is probably something that initially should be in pull request #43 
Will still keep it in here, as it is needed later. If the code runs through like this it is fine.